source/timesheet.js

```diff
@@ -19,23 +21,30 @@ class Timesheet{
         this.notes = timesheet.notes;
     }
 
-    save() {
+    async save() {
         return db.insertOne(this, collectionName);
     }
 
-    delete() {
+    async delete() {
+        await db.updateOne({_id : this.jobID}, { $pull: { timsheets : this.jobID} }, 'Jobs');
         return db.deleteOne({_id : this._id}, collectionName);
     }
 
```

Comment:
Please add a method comment for this method.
Should be in the form
Method Name
Parameters
- 
Returns
- 
Throws
-

Reply:
Added.